### ISSUE TITLE
Full list of polar bugs

### ISSUE THREAD

**Issue by etpinard** [plotly/plotly.js#275]
- radialAxis -> tickcolor doesn't do anything
- polar can't do categorical labels
- Closed loop charts with string coordinates shouldn't require `needsEndSpacing: false` ([example](http://plnkr.co/edit/4rVhmCmFeTnwi6jsmlmc?p=preview))

**Comment by louisza** [comment]
Setting showticklabels = FALSE in radialaxis does not work.

(R API)                      

```
library(plotly)
p <- plot_ly(plotly::wind, r = r, t = t, color = nms, type = "area")
layout(p, radialaxis = list(showticklabels = FALSE), orientation = 270)
```

**Comment by brian428** [comment]
Annotations don't work at all with Polar scatter plot.

**Comment by Kully** [comment]
don't want to open up a new issue until I know more:

In Python:
```
import plotly.plotly as py
import plotly.graph_objs as go
import numpy as np

trace = go.Scatter(
    r=[10],
    t=[0],
    name='adam',
    mode='markers',
    marker={'size': 80}
)

data = [trace]

fig = go.Figure(data=data)
py.iplot(fig)
```

results in the following:

![screen shot 2017-11-02 at 4 52 42 pm](https://user-images.githubusercontent.com/10369095/32349687-49adf224-bfee-11e7-873b-12543e941797.png)

A second and third trace appears automatically and the (t, r) coordinates (2, 11), (3, 12), and (4, 13) are automatically plotted.

@etpinard any idea where in `plotly.js` the error would be stemming from?

**Comment by etpinard** [comment]
fixed in `scatterpolar` after #2200 

see also https://github.com/plotly/plotly.js/issues/2255 for more polar development ideas
